includes/Withdraw/Hooks.php

```diff
@@ -72,4 +88,108 @@ public function update_vendor_balance( $withdraw ) {
             );
         }
     }
+
+    /**
+     * Handle withdraw request ajax.
+     *
+     * @since 3.3.1
+     *
+     * @return void
+     */
+    public function handle_withdraw_request() {
```

Comment:
since we are writing ajax callback to the same file, kindly add `ajax_` prefix to method name, this will increase readability